tasks/config/admin/templates/less-package.js

```diff
@@ -1,6 +1,5 @@
 let
-    where = 'client' // Adds files only to the client
-;
+    where = 'client'; // Adds files only to the client
```

Comment:
This should be merged with the previous line, so it's only

```javascript
let where = client;
```

In css-package.js, this was already done

Reply:
fixed